### PR TITLE
[WIP] - Vscode autocomplete

### LINKED ISSUE #5: vscode?
Support for vscode for autocomplete is on the cards, @paulmolluzzo are you back from that lovely vacation?

### PR DESCRIPTION
**Pull request by paulmolluzzo**
Fixes #5.

This is an early WIP. It's not very well documented, needs to be cleaned up, and doesn't support pseudoselectors. 😝 

I'll push some more changes over the next few days, but if anyone sees potential improvements please LMK. 🙏

### CODE REVIEW

File: packages/css-in-js-helpers/src/index.js [hunk 1]
```diff
@@ -25,9 +25,73 @@ const toCSS = item => {
...
+}
+
 export default function convert(s) {
   const lines = s.split('\n')
   return lines
     .map(item => (isCSS(item) ? toJS(item) : isJS(item) ? toCSS(item) : item))
     .join('\n')
 }
+
+export {
```
Comment (paulmolluzzo) [inline packages/css-in-js-helpers/src/index.js:82]: @ansumanshah many of these functions could be used in the atom version and dry up the codebase.

See: https://github.com/ansumanshah/css-in-js/blob/fa7b78d/packages/atom-css-in-js/lib/autocomplete.js#L6-L19

Comment (ansumanshah) [reply packages/css-in-js-helpers/src/index.js:82]: @paulmolluzzo true, will work on moving this as soon as we have the first alpha release for vscode ready 🎉

**Comment by paulmolluzzo** [comment]
@ansumanshah OK, so here's where I am with this PR, and I will need a little input to wrap it up:

* VS Code is working locally, but I haven't published any changes. I need to make a gif and update the README and package.json, but that should be fairly straightforward. The big question is if I can change the name of the extension and keep the users on the marketplace. :thinking: 
* Right now the scripts we have for the atom version only build for atom and are symlinked into vs code. I don't really like that, so are you up for moving those into the root project and writing the files into atom and vs code separately? I would also create a postinstall script in the root project that runs the scripts on install.
* A follow up point to that: The scripts are not ignored by git, so I'd have to commit changes on each run. Considering I want to put that into a `postinstall`, what do you think about ignoring the output? For vs code I think it's fine since it's the same as ignoring the `dist` directory.
* I haven't touched anything in the atom version, so all of that cleanup to reuse stuff from the helper package is left for a future PR.
* I'd like to flesh out the README for the main project to include instructions for building/contributing to each project. Sound good to you?

We're close! I can feel it! 🎉

**Comment by paulmolluzzo** [comment]
@ansumanshah This is published! 🎉 

https://marketplace.visualstudio.com/items?itemName=paulmolluzzo.convert-css-in-js

I'm merging this now since nothing should break the published packages for Atom or VS Code anyway. Any code cleanup to use the common `css-in-js-helpers` modules or bugs we find can be sorted out afterwards.

Thanks for your patience on this and sorry it took so long! 🙌  ...life 😅 

PS - if you can/want to add me as a contributor to the `css-in-js-helpers` package on npm I can published changes there too.

**Pull request merged**